varnishgather

```diff
@@ -610,18 +620,18 @@ for a in $(findmodsec); do
 done
 
 # maps
-for pid in $(pidof varnishd); do
+for pid in $(get_pid "varnishd"); do
 	runpipe "awk '$2 ~ \"rw\"' /proc/$pid/maps" "wc -l"
 done
 
 # Pick up the basic memory offset the text segments are in.
-for pid in $(pidof varnishd); do
+for pid in $(get_pid "varnishd"); do
 	run grep "r-xp" /proc/$pid/maps
 done
 
 # Get cache memory maps
-mycat /proc/$(pgrep cache-main)/smaps
-mycat /proc/$(pgrep cache-main)/status
+mycat /proc/$(get_pid "cache-main")/smaps
```

Comment:
what if `get_pid` returns multiple value?

Reply:
good catch! I'll test a bit more and will keep u posted.
Maybe we're facing the same issue using the current `pidof` as well?